lix/packages/sdk/src/open/openLix.ts

```diff
@@ -8,13 +8,31 @@ import { contentFromDatabase, createDialect, type SqliteDatabase } from "sqlite-
 /**
  * Common setup between different lix environments.
  */
-export async function openLix(args: { database: SqliteDatabase }) {
+export async function openLix(args: {
+	database: SqliteDatabase
+	/**
+	 * Usecase are lix apps that define their own file format,
+	 * like inlang (unlike a markdown, csv, or json plugin).
+	 *
+	 * (+) avoids separating app code from plugin code and
+	 *     resulting bundling logic.
+	 * (-) such a file format must always be opened with the
+	 *     file format sdk. the file is not portable
+	 *
+	 * @example
+	 *   const lix = await openLixInMemory({ blob: await newLixFile(), providePlugin: [myPlugin] })
+	 */
+	providePlugin?: LixPlugin[]
```

Comment:
why is it not called plugins?  pvovide seems confusing.

Reply:
will change it!